test.js

```diff
@@ -1,14 +1,20 @@
-
-
-document.getElementById('btn-menu').addEventListener('click', toggleMenuPopup)
-const ad = document.getElementById('links')
-const closeBtn = document.querySelector('.close-popup');
-
+const ad = document.getElementById('links');
 function toggleMenuPopup() {
   ad.classList.toggle('menu-popup');
 }
+document.getElementById('btn-menu').addEventListener('click', toggleMenuPopup);
+
+const closeBtn = document.querySelector('.close-popup');
 
 closeBtn.addEventListener('click', () => {
   ad.classList.remove('menu-popup');
-  document.body.classList.closeBtn('menu-open');
+  // document.body.classList.closeBtn('menu-open');
```

Comment:
- [ ] While you did the required changes by the previous reviewer, you left some places that need to be cleaned or fixed. Kindly remove unused commented codes from your project.